lumigator/python/mzai/backend/backend/services/jobs.py

```diff
@@ -36,6 +36,7 @@
 
 
 class JobService:
+    pass
```

Comment:
why did we add this?

Reply:
Sorry!!!! It was a trigger for the tests. I've just removed it.